File: Source/Core/Utility/NSNotification+Utility.m
//
//  NSNotification+Utility.m
//  Eunomia
//
//  Created by Ian Grossberg on 4/22/14.
//
//

#import "NSNotification+Utility.h"

@implementation NSNotification (Eunomia_Utility)

- (id)objectForInfoKey:(NSString *)key
{
    return [self.userInfo objectForKey:key];
}

@end

File: Source/Core/UtilityExtensions/NSNotification+Utility.swift
//
//  NSNotification+Utility.swift
//  Eunomia
//
//  Created by Ian Grossberg on 7/29/15.
//  Copyright (c) 2015 Adorkable. All rights reserved.
//

import Foundation

extension Notification {
    
    public func objectFromInfo(_ key : String) -> AnyObject? {
        return self.userInfo?[key] as AnyObject
    }
}
